lms/djangoapps/teams/tests/test_views.py

```diff
@@ -540,6 +542,10 @@ def test_full_student_creator(self):
         team_membership = team['membership']
         del team['membership']
 
+        # verify that it's been set to a time today.
+        self.assertEqual(parser.parse(team['last_activity_at']).date(), datetime.today().date())
```

Comment:
Does `datetime.today().date()` need to be timezone aware? `last_activity_at` is always being set with `datetime.utcnow().replace(tzinfo=pytz.utc)`.